lib/index.js

```diff
@@ -22,6 +25,27 @@ $('document').ready(function() {
   addFoodToMeal('snacks');
 });
 
+function displayDiaryFoods(foods) {
+  if(foods){
+    foods.forEach(function(food) {
+      appendRowToDiaryFoods(food["name"], food["calories"]);
+    });
+  }
+};
+
+function appendRowToDiaryFoods(name, calories) {
+  var markup = generateDiaryFoodRow(name, calories);
+  $('#add-food-list').prepend(markup);
+};
+
+function generateDiaryFoodRow(name, calories) {
+  return "<tr class='food-row'>" +
+            "<td><input type='checkbox' /></td>" +
+            "<td class='food-name food-field'>" + name + "</td>" +
+            "<td class='food-calories food-field right-align'>" + calories + "</td>" +
+          "</tr>"
```

Comment:
@CPowell23 It looks like we lost the delete button at some point. Can you please add it?